intl.emu

```diff
@@ -230,19 +231,19 @@ location: https://github.com/tc39/proposal-decimal/
         1. <ins>Let _m_ be *undefined*.</ins>
         1. <ins>Let _n_ be *undefined*.</ins>
         1. <ins>If _x_ is a Decimal128 value, then</ins>
-          1. <ins>Set _m_ be ! Decimal128ToDecimalString(_x_).</ins>
+          1. <ins>Set _m_ be Decimal128ToDecimalString(_x_).</ins>
           1. <ins>Set _f_ to the truncated exponent of _x_.</ins>
           1. <ins>Set _n_ to the scaled significand of _x_.</ins>
         1. Else,
           1. <del>Let</del><ins>Set</ins> _f_ <del>be</del><ins>to</ins> _maxFraction_.
-          1. [declared="n1"] Let _n1_ be an integer and _r1_ a mathematical value, with <emu-eqn>_r1_ = ToRawFixedFn(_n1_, _f_)</emu-eqn>, such that <emu-eqn>_n1_ modulo _roundingIncrement_ = 0</emu-eqn>, <emu-eqn>_r1_ ≤ _x_</emu-eqn>, and _r1_ is maximized.
+          1. [declared="n1,r1"] Let _n1_ be an integer and _r1_ a mathematical value, with <emu-eqn>_r1_ = ToRawFixedFn(_n1_, _f_)</emu-eqn>, such that <emu-eqn>_n1_ modulo _roundingIncrement_ = 0</emu-eqn>, <emu-eqn>_r1_ ≤ _x_</emu-eqn>, and _r1_ is maximized.
           1. [declared="n2,r2"] Let _n2_ be an integer and _r2_ a mathematical value, with <emu-eqn>_r2_ = ToRawFixedFn(_n2_, _f_)</emu-eqn>, such that <emu-eqn>_n2_ modulo _roundingIncrement_ = 0</emu-eqn>, <emu-eqn>_r2_ ≥ _x_</emu-eqn>, and _r2_ is minimized.
           1. Let _r_ be ApplyUnsignedRoundingMode(_x_, _r1_, _r2_, _unsignedRoundingMode_).
           1. If _r_ is _r1_, then
-            1. Let _n_ be _n1_.
+            1. <ins>Set</ins> _n_ <del>be</del><ins>to</ins> _n1_.
             1. Let _xFinal_ be _r1_.
           1. Else,
-            1. Let _n_ be _n2_.
+            1. <del>Let</del><ins>Set</ins> _n_ <del>be</del><ins>to</ins> _n2_.
```

Comment:
I'd suggest trying to get these fixes that are not specific to Decimal, already into the ECMA-402 text. For now, it's probably clearer to just edit it to be "Set _n_ to _n1_" etc., without the `<ins>` and `<del>` tags. Otherwise it's confusing to the reader why this change is part of Decimal.

Reply:
Sure, I can do that. I've simplified the change to just do "Set ...", as suggested.